compiler/resolution/wrappers.cpp

```diff
@@ -1637,10 +1665,7 @@ static void handleInIntent(FnSymbol* fn, CallExpr* call,
           tmp->addFlag(FLAG_CONST_DUE_TO_TASK_FORALL_INTENT);
         }
 
-        bool isConstCopy = formal->intent == INTENT_CONST_IN ||
-                           formal->originalIntent == INTENT_CONST_IN ||
-                           formal->hasFlag(FLAG_CONST_DUE_TO_TASK_FORALL_INTENT);
-        Symbol *definedConst = isConstCopy ?  gTrue : gFalse;
+        Symbol *definedConst = isFormalTempConst(fn, formal) ?  gTrue : gFalse;
```

Comment:
Not a big deal but it seems we could reuse `isFormalTempConst` in both branches by storing it in a variable above

Reply:
Right, but after all this testing and this close to code freeze, I am a bit reluctant to make that change :)